hetzner/README-cloud.md

```diff
@@ -91,6 +96,29 @@ It's used so Ansible can connect to the server without requiring password.
 
 # The VM
 
+## All-in-one creation
+
+2 new playbooks have been created to aggregate the different operations executed by the different playbooks
+, `hetzner-create-server-aggregate.yml` and `hetzner-delete-server-aggregate.yml`.
+
+To create and deploy a new VM simply execute the following command on the root of the `k8s-infra` project.
+
+```bash
+$ ansible-playbook hetzner/ansible/hetzner-create-server-aggregate.yml -e vm_name=my_vm -e k8s_type=masters -e k8s_version=119 -e salt_text=$(gpg --gen-random --armor 1 20)  --tags create
+```
+
+| Variable | Required | Default | Prompt | Meaning |
+| --- | :---: | :---: | :---: | --- |
+| k8s_type | | | | Type of k8s node (masters, nodes). Will be used to set the ansible inventory groups. |
+| k8s_version | | | | K8s version to be later installed. Will be used to set the ansible inventory groups. |
+| salt_text | x | | | Salt and pepper. |
+| vm_name | x |  | | Name of the VM to be craated. This will be both the name on Hetzner as well as on the inventory |
+
+Removing a VM also has an aggregate playbook, requiring only the `vm_name` parameter.
+
+```bash
+$ ansible-playbook hetzner/ansible/hetzner-delete-server-aggregate.yml -e vm_name=my_vm
```

Comment:
Why dont you use as playbook name `delete-vm-and-k8s.yml` ?

Reply:
because this playbook only deletes the VM server from hetzner, it doesn't execute any k8s roles/tasks.

Reply:
ok